SwiftLint/swiftlint.yml

```diff
@@ -54,6 +55,39 @@ opt_in_rules:
   - identical_operands
   - collection_alignment
   - empty_xctest_method
+  - array_init
+  - contains_over_filter_count
+  - contains_over_filter_is_empty
+  - contains_over_first_not_nil
+  - empty_collection_literal
+  - empty_string
+  - explicit_self
+  - file_name
+  - for_where
+  - joined_default_parameter
+  - last_where
+  - legacy_multiple
+  - literal_expression_end_indentation
+  - lower_acl_than_parent
+  - modifier_order
+  - multiline_arguments
+  - multiline_function_chains
+  - multiline_literal_brackets
+  - multiline_parameters_brackets
+  - object_literal
+  - overridden_super_call
+  - redundant_type_annotation
+  - sorted_first_last
+  - static_operator
+  - toggle_bool
+  - unowned_variable_capture
+  - unused_import
+  - vertical_parameter_alignment_on_call
+  - vertical_whitespace_closing_braces
+  - convenience_type
+  - number_separator
```

Comment:
This creates a lot of warnings on HiDrive. Sometimes weird situations, for example for a color declaration
- `UIColor(red: 0.9294117647, green: 0.9294117647, blue: 0.9294117647, alpha: 1.0)` is triggered for example.
Overall, nice to have but maybe for later ?!